docs/reference/guard_get_installer.md

```diff
@@ -39,7 +39,8 @@ guard get installer [flags]
       --azure.resource-id  string            azure cluster resource id (/subscriptions/<SubscriptionId>/resourceGroups/<RGname>/providers/Microsoft.ContainerService/managedClusters/<clustername> for AKS or /subscriptions/<SubscriptionId>/resourceGroups/<RGname>/providers/Microsoft.Kubernetes/connectedClusters/<clustername> for arc) to be used as scope for RBAC check
       --azure.skip-authz-check strings       comma separated list of user email ids for which Azure RBAC will be skipped. (default empty)
       --azure.skip-authz-for-non-aad-users   If true, Azure RBAC will be skipped for certificate users (non AAD users). Setting to false, all non AAD user requests will be denied. Azure RBAC will always be skipped for service accounts. This setting does not affect Azure RBAC behavior for service accounts. (default true)
-      --azure.authz-resolve-group-memberships If true, authroizer will resolve group membership. Setting to false will use group list from subjectaccessreview request. (default true)
+      --azure.graph-call-on-overage-claim    If true, authorizer will resolve group membership only when overage claim is present. Setting to false will always call graph api to resolve group membership (default true)
+      --azure.authz-resolve-group-memberships If true, authroizer will resolve group membership. Setting to false will use group list from subjectaccessreview request. (default false)
```

Comment:
please add that we should set it false always. Also I think we should fail deployment if it's set as true to avoid this to be used.
@weinong what do you think?

Reply:
you never know who else is using guard. so lets not fail it.

Reply:
actually i take that back, since you are proposing changing under authz, which is relatively new. I'm totally fine with Krupesh's proposal